app/pages/event1/member/ReasonForTheOverpaymentOrWriteOffPage.scala

```diff
@@ -30,4 +32,11 @@ case object ReasonForTheOverpaymentOrWriteOffPage extends QuestionPage[ReasonFor
 
   override def route(waypoints: Waypoints): Call =
     routes.ReasonForTheOverpaymentOrWriteOffController.onPageLoad(waypoints)
+
+  override protected def nextPageNormalMode(waypoints: Waypoints, answers: UserAnswers): Page = {
+    answers.get(ReasonForTheOverpaymentOrWriteOffPage) match {
```

Comment:
afaik, this page has radio buttons. if selection is mandatory, we can remove again matching and forwarding directly PaymentValueAndDatePage

Reply:
Resolved in next commit